examples/hello_world/config/config_translator.textproto

```diff
@@ -1,19 +0,0 @@
-node_configs {
```

Comment:
Why is this removed?  It seems to remove the ability to run `hello_world` with a `translator` module configured.